espnet/nets/pytorch_backend/e2e_tts_transformer.py

```diff
@@ -202,6 +202,8 @@ def add_arguments(parser):
                            help="Dropout rate in postnet")
         group.add_argument("--pretrained-model", default=None, type=str,
                            help="Pretrained model path")
+        group.add_argument("--train-embedding", default=True, type=strtobool,
```

Comment:
How about moving this option to `tts_train.py`?
My suggestion is to make it similar to following options:
https://github.com/espnet/espnet/blob/043baca91d461022b1f3ae04f3152890aab9b830/espnet/bin/tts_train.py#L126-L130
For example, `--freeze-mods` which recieves comma separeted module names to be freezed (e.g. `freeze-mods: encoder.embed`).
And then you can set `require_grad = False` before optimizer setup, around here.
https://github.com/espnet/espnet/blob/043baca91d461022b1f3ae04f3152890aab9b830/espnet/tts/pytorch_backend/tts.py#L325-L328
Then, we can use freeze function for arbitrary models.